README.md

```diff
@@ -1,157 +1,160 @@
-<img src="https://www.creditas.com.br/static/images/logo-creditas-color-8367919c2a.svg" width="400">
+<img src="./logo-creditas.svg" width="400">
 
-# Frontend Challenge - Creditas
+# Frontend Challenge
 
 This is a challenge designed to measure the candidate's expertise.
-We would like to clarify that we don't expect everyone to be able to finish the whole challenge, since it has been designed to cover several degrees of expertise, from junior to senior.
-Nevertheless, we expect anyone interested in becoming a tripulante in Creditas to give it a try.
 
-In this challenge,cd we focus both on code design and design patterns in JavaScript. The goal is to assess your experience in writing **code that is easy to maintain, loosely coupled and highly cohesive**.
+In this challenge,we focus both on code design and design patterns in JavaScript.
 
-At Creditas we always work giving constructive feedback, therefore we will always dedicate the outmost attention to every solution submitted; contacting you back with the positive aspects of your submission and what could be improved. Just for that it is worth trying! :)
+At Creditas we always work giving constructive feedback, therefore we will always dedicate the outmost attention to every solution submitted, contacting you back with the positive aspects of your submission and what could be improved.
+
+Just for that it is worth trying! :)
 
 ## The challenge
 
-You have to improve the loan simulator implemented in this project.
+Cine Creditas is getting bigger and bigger and needs to create a self-service booking system.
 
-Both the interface as well as the styles are provided. The challenges include refactoring the code and implementing new functionalities (listed below).
+In order to make a booking, users must select a **film** and a **snack**.
 
-Feel free to componetize whatever you think should be componetized. Our only request is that you use only pure JavaScript, our beloved Vanilla. Do you accept the challenge?
+It is **important** to remark that our business model requires the user to select both elements ( a film and a snack )
 
-This is the simulator current state:
+Once the user has selected both items, they must confirm their selection and as a result a ticket will be displayed describing user's choice.
 
-<img style="display: block; margin: 0 auto;" src="./layout_en.png">
+Your mission will be to build a web application where the user can see both a list of films and a list of snacks, have the possibility to confirm their selection and display the corresponding generated ticket.
 
+Right now due to the traffic we are generating, we are averaging over 10 seconds to generate the ticket.
 
-### Expected functionalities
+As part of the challenge we need your help in order to reduce this delay as much as possible.
```

Comment:
Could you please provide further explanation about the reasoning of these criteria? 

I'm struggling to understand what exactly the candidate has to do in order to improve this 10 seconds delay. Since there is no additional code/implementation in the assignment, it might be super easy for the candidate to pass into these criteria, so I'm not sure if its worth to add this here

is there some kind of delay in the `POST: /api/films/confirm` and `POST: /api/snacks/confirm`?

Reply:
@rsicarelli The thing is that the ticket needs information from the response of both endpoints. They can do these requests sequentially or in parallel. This is where the time will considerably change, since the requests take 5 seconds to respond.

Reply:
To clarify:

> is there some kind of delay in the POST: /api/films/confirm and POST: /api/snacks/confirm?

Yes, there is!

Reply:
Awesome, clear! You can resolve this comment if you wish 👌